File: Competitive Coding - 1/1. Problems on Array/Q2.c
#include <string.h>
int sumOfHighestAndLowestElements(int arr[], int arrLen) {
    // Write from code here
    int min=arr[0],max=arr[0];
    for(int i=1;i<arrLen;i++){
        if(arr[i]>max)
            max=arr[i];
        if(arr[i]<min)
            min=arr[i];
    }
        return max+min;
    // ends here
}
int readIntArray(char *argsArray, int arr[]) {
    int col = 0;
    char *token = strtok(argsArray, ",");
    while (token != NULL) {
        arr[col] = atoi(token);
        token = strtok(NULL, ",");
        col++;
    }
    return col;
}
int main(int argc, char *argv[]) {
    int arr[strlen(argv[1])];
    int arrLen = readIntArray(argv[1], arr);
    printf("%d\n", sumOfHighestAndLowestElements(arr, arrLen));
    return 0;
}

File: Competitive Coding - 1/2. Problems on Strings/Q2.py
import sys

s = sys.argv[1]

def isPrimeLengthPalindrome(s):
	# write code from here
	n=len(s)
	rev=""
	flag=0
	if n>1:
		for i in range(2,n):
			if n%i==0:
				flag=1
				break
	if flag==1:
		return False
	else:
		for i in s:
			rev=i+rev
		if rev==s and flag==0:
			return True
		else:
			return False
	# ends here

print("true" if isPrimeLengthPalindrome(s) else "false");
